nautobot/extras/utils.py

```diff
@@ -29,7 +28,6 @@
 logger = logging.getLogger(__name__)
 
 
-@file_cache.cached(timeout=60)
```

Comment:
Since we aren't removing any other prefetches, etc. is there a special reason to stop these cache entries now?

Reply:
In my testing to compare solutions, I was not able to reproduce the original issue with cascading calls in the job results list view. As far as I can tell, our refactoring of `Job` internals with the concrete model work made this problem go away and therefore the file caching is no longer necessary.